content/lessons/chapter-3/you-vs-bit-1.tsx

```diff
@@ -0,0 +1,22 @@
+'use client'
+
+import { useTranslations } from 'hooks'
+import { Introduction, Text } from 'ui'
+
+export const metadata = {
+  title: 'chapter_two.intro_one.title',
+  image: '/assets/images/chapter-2-intro-1.jpg',
+  key: 'CH3YVB1',
```

Comment:
Keys can be `CH3SOL1`, `CH3COO1`, `CH3POO1` (that last one is a but unfortunate lol)

Reply:
Addressed in https://github.com/saving-satoshi/saving-satoshi/commit/5a1178b3952e7d024452fad6416a2a65c9fdba47